packages/js/e2e-utils-playwright/src/pdp.js

```diff
@@ -0,0 +1,24 @@
+const { expect } = require( '@playwright/test' );
+
+/**
+ * Util helper made for adding multiple same products to cart
+ *
+ * @param page
+ * @param productName
+ * @param quantityCount
+ */
+export async function addProductsToCart( page, productName, quantityCount ) {
```

Comment:
I'm not crazy about the name of this function.  `addOneOrMoreProduct()` might be better?  That way it can't be confused that it adds different products.